utils/README.md

```diff
@@ -63,3 +66,87 @@ To install and configure a system with Kata Containers and containerd, run:
 ```bash
 $ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh)"
 ```
+
+### Choose a Hypervisor
+
+Kata works with different [hypervisors](../docs/hypervisors.md). When you install a Kata system, the default hypervisor
```

Comment:
I understand the motivation on linking to the hypervisors page, but the user will not know which name to use to select, for instance, "Cloud Hypervisor".

Reply:
We need a way to have the list of supported hypervisors being displayed without installing kata.

Reply:
It's not explained in the commits, but I tried to minimise what we hard-code in the script as it's better to rely on the structure of the packaged config files to determine the logic. Further, it will simplify our lives when we tackle #8677 and #9060.

 I've now updated the branch so that the `hypervisors.md` page shows the short names, which allows the users to determine the names used by the script.